src/components/login/SignIn.tsx

```diff
@@ -22,8 +29,26 @@ export const SignIn = ({ changeMode }: SignInProps) => {
   const [password, setPassword] = useState('');
   const [error, setError] = useState('');
   const [isLoading, setLoading] = useState(false);
+  const [visible, setVisible] = useState('password');
   const { authenticate } = useAuth();
 
+  const toast = useToast();
+
+  const toastErrorTemplate = () => {
+    const id = 'uniqueToast';
```

Comment:
id should be an random string. use [uuid](https://www.npmjs.com/package/uuid) or [cuid](https://www.npmjs.com/package/cuid) lib to generate it!

Reply:
ok, i will do it